src/rich-text/commands/index.ts

```diff
@@ -37,7 +37,7 @@ import { _t } from "../../shared/localization";
 export * from "./tables";
 
 //TODO
-function toggleWrapIn(nodeType: NodeType) {
+export function toggleWrapIn(nodeType: NodeType) {
```

Comment:
needed to test blockquote command which is generated via this method.  Other commands (like `toggleHeadingLevel`) are exported from here as well, so no over-arching reason to keep private

Reply:
If you expose this method, you need to write documentation and unit tests for it 😉

Reply:
so, in a sense, this is actually the system under test when testing blockquotes can be inserted into lists.  What we want to check is a) can we execute this command and b) does it apply the right transform.  So there's a single existing test for it, but I added some more test cases for some basic coverage of toggling back and forth